nixos/modules/services/games/teeworlds.nix

```
@@ -383,7 +410,15 @@ in
 
       serviceConfig = {
         DynamicUser = true;
-        ExecStart = "${cfg.package}/bin/teeworlds_srv -f ${teeworldsConf}";
+        RuntimeDirectory = "teeworlds";
+        RuntimeDirectoryMode = "0700";
+        EnvironmentFile = lib.mkIf (cfg.environmentFile != null) cfg.environmentFile;
+        ExecStartPre = ''
+          ${pkgs.envsubst}/bin/envsubst \
+            -i ${teeworldsConf} \
+            -o /run/teeworlds/teeworlds.yaml
+        '';
+        ExecStart = "${cfg.package}/bin/teeworlds_srv -f /run/teeworlds/teeworlds.yaml";
```

While unrelated to the actual PR, maybe we can use `lib.getExe` here:
```suggestion
        ExecStart = "${lib.getExe cfg.package} -f /run/teeworlds/teeworlds.yaml";
```

---

Added new commit with this change :)